akka-actor/src/main/scala/akka/japi/function/Function.scala

```diff
@@ -23,6 +31,16 @@ trait Function2[-T1, -T2, +R] extends java.io.Serializable {
   def apply(arg1: T1, arg2: T2): R
 }
 
+object Function2 {
```

Comment:
That's not *mandatory* for this PR, but it allows to write more simply code like in `Keep`:

```scala
// Before
private val _left = new function.Function2[Any, Any, Any] with ((Any, Any) ⇒ Any) { def apply(l: Any, r: Any) = l }

// After
private val _left = Function2[Any, Any, Any] { (l: Any, _: Any) ⇒ l }
```